File: 26_PracticeProblem2.cpp
/* Write a C++ program to

create an Employee class with data members: 

Employee number, Employee name, Basic, DA, IT, Net Salary

class has 3 methods, one method to get employee details for name, ID and basic salary

other method to calculate net salary using formula

	DA=1.32*basic;

	IT=0.30*(basic+DA);

	net_sal=(basic+DA)-IT; 

third method to print emp nam*/
#include<iostream>
using namespace std;

class Employee
{
	public:
	//Delaring attributes
	int Employee_number;
	string Employee_name;
	float Basic;
	float DA;
	float IT;
	float Salary;
	
	void get_details()             
	{
		cout<<"Enter Employee ID"<<endl;
		cin>>Employee_number;
		cout<<"Enter Employee name "<<endl;
		cin>>Employee_name;
		cout<<"Enter Employee Basic Salary"<<endl;
		cin>>Basic;
	}
	void calculate_details()       
	{
		DA=1.32*Basic;
		IT=0.30*(Basic+DA);
		Salary=(Basic+DA)-IT;	
	}
	void print_details()           
	{
		cout<<"Employ Id"<<Employee_number<<endl;
		cout<<"Employ name : "<<Employee_name<<endl;
		cout<<"Employ Basic Salary : "<<Basic<<endl;
		cout<<"Employ DA : "<<DA<<endl;
		cout<<"Employ IT : "<<IT<<endl;
		cout<<"Employ Salary : "<<Salary<<endl;
	}
};
   
    int main()
    {
    Employee E1[1];  //Creating array of objects
    for(int i=0 ; i<1; i++)
    {
    	cout<<"Enter details of Employee:"<<i+1<<endl;
    	E1[i].get_details();
    	E1[i].calculate_details();
        E1[i].print_details();
	}
    return 0;
}

File: 26_PracticeProblem2.java
import java.util.Scanner;
public class Employee {                                //Class Employee
    Scanner in = new Scanner(System.in);             
    //Declaring Variables
    int Employ_number;    
    String Employ_name;
    double Basic;
    double DA;
    double IT;
    double Salary;
    public Employee(){

    }

    void get_details() {                                       //Method to get details
        System.out.println("Enter the Id of Employee : ");
        Employ_number = in.nextInt();
        System.out.println("Enter the name of Employee : ");
        Employ_name = in.next();
        System.out.println("Enter the Basic Salary of Employee");
        Basic = in.nextFloat();
    }

    void calculate_details() {                                       //Method to Calculate details
        DA = 1.32 * (Basic);
        IT = 0.30 * (Basic + DA);
        Salary = (Basic + DA) - IT;
    }

    void print_details() {                                           //Method to find details
        System.out.println("Employ Id" + Employ_number);
        System.out.println("Employ name : " + Employ_name);
        System.out.println("Employ Basic Salary : " + Basic);
        System.out.println("Employ DA : " + DA);
        System.out.println("Employ IT : " + IT);
        System.out.println("Employ Salary : " + Salary);
    }

    public static void main(String[] args) {
    	Employee[] obj = new Employee[2];                           //Array of object to take Multiple details
        for (int i = 0; i < 2; i++) {
            System.out.println("Enter details of Employee:"+(i+1));
            obj[i]= new Employee();
            obj[i].get_details();
            obj[i].calculate_details();
        }
        for (int i = 0; i < 2; i++) {
            obj[i].print_details();
        }
    }

}
